freud/environment.pyx

```diff
@@ -340,18 +339,20 @@ cdef class LocalDescriptors(Compute):
             A reference to the last computed spherical harmonic array.
         num_particles (unsigned int):
             The number of points passed to the last call to :meth:`~.compute`.
-        num_neighbors (unsigned int):
-            The number of neighbors used by the last call to compute. Bounded
-            from above by the number of reference points multiplied by the
-            lower of the num_neighbors arguments passed to the last compute
-            call or the constructor.
+        num_sphs (unsigned int):
+            The last number of spherical harmonics computed. This is equal to
+            the number of bonds in the last computation, which is bounded from
+            above by the number of reference points multiplied by the lower of
```

Comment:
Fix "reference points" to new nomenclature. Also I'd prefer to use less mathematical language here:
"which is at most the number of points times ..."
 ...
"(it may be less if there are not enough neighbors for every particle)."